src/ntp64.rs

```diff
@@ -96,6 +96,12 @@ impl NTP64 {
         (self.0 >> 32) as u32
     }
 
+    /// Returns the total number of nanoseconds.
+    #[inline]
+    pub fn as_nanos(&self) -> u64 {
+        self.to_duration().as_nanos() as u64
```

Comment:
Could it be more performant to skip the conversion and allocation to Duration?

Reply:
Done